templates/field.php

```diff
@@ -48,6 +48,10 @@
 		</div>
 	</script>
 
-	<div class="zone-posts-list"></div>
+	<div class="zone-posts-list">
+	</div>
 
-</div>
\ No newline at end of file
+	<?php foreach ( $current_ids as $id ) : ?>
+		<input type="hidden" class="fm-element fmz-remove-if-js" name="<?php echo esc_attr( $this->get_form_name( '[]' ) ); ?>" value="<?php echo absint( $id ) ?>" />
```

Comment:
WP Coding standards doesn't take a position on semicolons in inline statements like this -- in fact, the coding standards page has examples both with and without them. We can ignore these in code reviews.